api/src/test/java/org/openmrs/module/insuranceclaims/InsuranceClaimsServiceTest.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.insuranceclaims.api;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.openmrs.User;
+import org.openmrs.api.UserService;
+import org.openmrs.module.insuranceclaims.Item;
+import org.openmrs.module.insuranceclaims.api.dao.InsuranceClaimsDao;
+import org.openmrs.module.insuranceclaims.api.impl.InsuranceClaimsServiceImpl;
+
+import static org.hamcrest.Matchers.hasProperty;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.when;
+
+/**
+ * This is a unit test, which verifies logic in InsuranceClaimsService. It doesn't extend
+ * BaseModuleContextSensitiveTest, thus it is run without the in-memory DB and Spring context.
+ */
+public class InsuranceClaimsServiceTest {
+	
+	@InjectMocks
+	InsuranceClaimsServiceImpl basicModuleService;
+	
+	@Mock
+	InsuranceClaimsDao dao;
+	
+	@Mock
+	UserService userService;
+	
+	@Before
+	public void setupMocks() {
+		MockitoAnnotations.initMocks(this);
+	}
+	
+	@Test
+	public void saveItem_shouldSetOwnerIfNotSet() {
+		//Given
+		Item item = new Item();
+		item.setDescription("some description");
+		
+		when(dao.saveItem(item)).thenReturn(item);
+		
+		User user = new User();
+		when(userService.getUser(1)).thenReturn(user);
```

Comment:
what does it mean userService.getUser(1) ? Why 1 ?

Reply:
This tested the generated service which if needed set the owner as a user with id 1 (probably the admin user). We don't need that logic so I will change that in the separate PR where I will add required POJO classes.